observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -46,7 +46,19 @@ include::snippets/technology-preview.adoc[leveloffset=+1]
 
 This update introduces the following enhancements:
 
-* ???.
+// https://issues.redhat.com/browse/TRACING-4108
+* Added support for securing Jaeger UI and Jaeger APIs with OpenShift Oauth proxy.
+// https://issues.redhat.com/browse/TRACING-3954
+* Added support for using OpenShift serving certificates on ingestion APIs when multitenancy is disabled.
+// https://issues.redhat.com/browse/TRACING-4171
+* Added support for ingesting OpenTelemetry protocol (OTLP) JSON when multitenancy is enabled.
+// https://issues.redhat.com/browse/TRACING-4176
+* Added support for AWS S3 Secure Token authentication.
+// https://issues.redhat.com/browse/TRACING-4185
+* Added support for automatically reloading certificates.
+// https://issues.redhat.com/browse/TRACING-4214
```

Comment:
We should improve the description of that issue

Reply:
I've copied your comment into https://github.com/openshift/openshift-docs/pull/80135/files#r1723385025.